# So we don't have to ship that massive ugly beast of the moz/ directory.

diff -u -p -u -r1.18 build.lst
--- connectivity/prj/build.lst	2 Jun 2003 07:53:47 -0000	1.18
+++ connectivity/prj/build.lst	17 Jul 2003 15:17:05 -0000
@@ -1,4 +1,4 @@
-cn	connectivity	:	comphelper moz svtools unixODBC NULL
+cn	connectivity	:	comphelper svtools unixODBC NULL
 cn	connectivity							usr1	-	all	cn_mkout NULL
 cn	connectivity\source\types				nmake	-	all	cn_types NULL
 cn	connectivity\source\commontools			nmake	-	all	cn_cmtools cn_parse cn_types NULL
